### PR TITLE
Fix command bar guide colors

### PR DESCRIPTION
**Pull request by Ved235**
## The problem
Initially the guide text colors in the command bar didn't switch correctly with change in theme causing difficulty in reading text.

## Changes

**Initial dark theme view**
image

**Final dark theme view**
image

_Working for light theme view as well_
image
image

### CODE REVIEW

File: app/javascript/components/command_bar/search/guide.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
-/* eslint react/prop-types:0 */
```
Comment (sampoder) [inline app/javascript/components/command_bar/search/guide.js]: Let's not remove this if possible

File: app/javascript/components/command_bar/search/guide.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-
 import React from 'react'
 
 export function Guide() {
```
Comment (sampoder) [inline app/javascript/components/command_bar/search/guide.js]: Would it be possible to wrap this in one span? Does that work?

**Comment by Ved235** [comment]
Hi, I made a new commit to resolve these.

**Review by sampoder — DISMISSED** [review]
LGTM!

**Comment by sampoder** [comment]
Thanks helping fix this, I really appreciate that!

**Pull request merged**